FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -5592,14 +5593,14 @@ void OpDispatchBuilder::TZCNT(OpcodeArgs) {
 void OpDispatchBuilder::LZCNT(OpcodeArgs) {
   OrderedNode *Src = LoadSource(GPRClass, Op, Op->Src[0], Op->Flags, -1);
 
-  auto Res = _CountLeadingZeroes(Src);
+  auto Res = _CountLeadingZeroes(IR::SizeToOpSize(GetSrcSize(Op)), Src);
   StoreResult(GPRClass, Op, Res, -1);
   GenerateFlags_LZCNT(Op, Src);
 }
 
 void OpDispatchBuilder::MOVBEOp(OpcodeArgs) {
   OrderedNode *Src = LoadSource(GPRClass, Op, Op->Src[0], Op->Flags, 1);
-  Src = _Rev(Src);
+  Src = _Rev(IR::SizeToOpSize(GetSrcSize(Op)), Src);
```

Comment:
Can we get a helper for `IR::SizeToOpSize(GetSrcSize(Op))`? It seems to clutter things.

Maybe `IR::OpSizeFromSrc(Op)`?

Reply:
Couldn't add it to `IR::` namespace since `X86Tables::DecodedOp` and IR operation sizes are legally distinct entities, but added the helper to `OpcodeDispatcher` so we can use it all in there.

Reply:
Sounds good, thanks